webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/ViewTechnique.elm

```diff
@@ -60,6 +61,48 @@ showTechnique model technique origin ui =
                  Creation _ -> True
                  Clone _ _ -> True
                  Edit _ -> False
+    methodCallList = List.concatMap getAllCalls technique.elems
+
+    {-| For each MethodCall in techniques we compute each methods's parameter
+       For exemple :
+          In technique "foo" there is two methods
+           * "command_execution" with id "e8a8d662-9d2d-44b4-87f8-591fb6f4db1b"
+             - [Mandatory] with parameter "Command"       -> empty
+           * "package_present" with id "6394770e-a9c2-4e67-8569-d21560a07dc2
+             - [Mandatory] with parameter "Name"         -> empty
+             - [Optional] with parameter  "Version"      -> "latest"
+             - [Optional] with parameter  "Architecture" -> empty
+             - [Optional] with parameter  "Provider"     -> "apt"
+
+       the result will be :
+       [
+         ( "e8a8d662-9d2d-44b4-87f8-591fb6f4db1b"
+         , InvalidState [ConstraintError { id = { value = "command" }, message = "Parameter 'command' is empty" }])
+         ) ,
+         ( "e8a8d662-9d2d-44b4-87f8-591fb6f4db1b"
```

Comment:
shouldn' it be 6394770e-a9c2-4e67-8569-d21560a07dc2 and parameter Name is empty ?